docs/riff_streaming_stream_create.md

```diff
@@ -0,0 +1,44 @@
+---
+id: riff-streaming-stream-create
+title: "riff streaming stream create"
+---
+## riff streaming stream create
+
+create a stream of messages
+
+### Synopsis
+
+<todo>
+
+```
+riff streaming stream create <name> [flags]
+```
+
+### Examples
+
+```
+riff streaming stream create --provider my-provider
+```
+
+### Options
+
+```
+      --content-type MIME type   MIME type for message payloads accepted by the stream
+      --dry-run                  print kubernetes resources to stdout rather than apply them to the cluster, messages normally on stdout will be sent to stderr
+  -h, --help                     help for create
+  -n, --namespace name           kubernetes namespace (defaulted from kube config)
+      --provider name            name of stream provider
```

Comment:
isn't `--provider` = the name of the `kafka-provider` with the additional suffix: `-kafka-provisioner` ?

(i'd prefer what's in the help now but we should document reality :))

Reply:
this was a point of debate during the creating of the provider resource. In general, I didn't touch docs that had already been written. I'll let @ericbottard take a pass at the doc content.